src/tasks/leveling.ts

```diff
@@ -399,46 +459,44 @@ export const LevelingQuest: Quest = {
       name: "Wish for XP% buff",
       // TODO: Make this completed if we've already wished twice with the paw (requires mafia tracking)
       completed: () =>
-        have($effect`Different Way of Seeing Things`) ||
+        have(xpWishEffect)  ||
         !have($item`cursed monkey's paw`) ||
-        forbiddenEffects.includes($effect`Different Way of Seeing Things`) ||
+        forbiddenEffects.includes(xpWishEffect) ||
         get("instant_saveMonkeysPaw", false) ||
-        myBasestat($stat`Mysticality`) >= targetBaseMyst - targetBaseMystGap ||
+        myBasestat(myPrimestat()) >= targetBaseMainStat - targetBaseMainStatGap || /*Dearest Seraphiii - Update this later*/
```

Comment:
If this is fixed, we can remove the comment (or add it to a `TODO`)

Reply:
I added the other two mainstat 'Gazes' - they're not as powerful, or as good, but they do exist. Smell benefits are still benefits.

Reply:
Wouldn't +25% mainstat be better than +1 flatxp?